VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-status/participant-status.component.spec.ts

```diff
@@ -264,4 +265,30 @@ describe('ParticipantStatusComponent', () => {
             expect(videoWebServiceSpy.updateParticipantDisplayName).toHaveBeenCalled();
         });
     });
+    describe('Delete quick link disconnected participant', () => {
+        it('should delete participant, when delete button clicked', () => {
+            videoWebServiceSpy.deleteParticipant.and.returnValue(Promise.resolve());
+            component.conferenceId = '123';
+            const participantQuickLinkDisconnected = new ParticipantContactDetails(participants[5]);
+            component.deleteParticipant(participantQuickLinkDisconnected);
+            expect(videoWebServiceSpy.deleteParticipant).toHaveBeenCalledWith('123', participantQuickLinkDisconnected.id);
+        });
+
+        it('should log error when delete quick link participant', () => {
+            const error = new Error('Failed to delete participant');
+            videoWebServiceSpy.deleteParticipant.and.returnValue(Promise.reject(error));
+            component.conferenceId = '123';
+            const participantQuickLinkDisconnected = new ParticipantContactDetails(participants[5]);
+            component.deleteParticipant(participantQuickLinkDisconnected);
+            expect(videoWebServiceSpy.deleteParticipant).toHaveBeenCalled();
+        });
+
+        it('should return true when participant is deletable', () => {
```

Comment:
I would suggest expanding these two tests to cover the different combinations of participant role and status. Also, creating a participant object rather than relying on the shared test data.

something like:

```
describe('isParticipantDeletable', () => {
        it('should return true if the participant is a QuickLinkParticipant and is Disconnected', () => {
            const participant = new ParticipantContactDetailsResponseVho({
                role: Role.QuickLinkParticipant,
                status: ParticipantStatus.Disconnected
            });

            expect(component.isParticipantDeletable(participant)).toBeTrue();
        });

        it('should return true if the participant is a QuickLinkObserver and is Disconnected', () => {
            // TODO
        });
    
        it('should return false if the participant is a QuickLinkParticipant but is not Disconnected', () => {
            // TODO
        });
    
        it('should return false if the participant is a QuickLinkObserver but is not Disconnected', () => {
            // TODO
        });
    
        it('should return false if the participant role is not QuickLinkParticipant or QuickLinkObserver, even if Disconnected', () => {
            // TODO
        });
    });
```

Reply:
added tests